onnxruntime/core/mlas/lib/sqnbitgemm.h

```diff
@@ -235,4 +326,14 @@ struct MLAS_SQNBIT_GEMM_DISPATCH {
     );
 
     QuantizeARow_CompInt8_Fn* QuantizeARow_CompInt8 = nullptr;
+
+    typedef void(QuantizeARow_CompInt8_Fn2)(
```

Comment:
>QuantizeARow_CompInt8_Fn2 [](http://example.com/codeflow?start=17&length=25)

a better name

Reply:
changed to QuantizeARowComputeBlkSum_CompInt8